### ISSUE TITLE
Broken build on OSX?

### ISSUE THREAD

**Issue by pelson** [conda-forge/m4-feedstock#10]
```
> conda list
# packages in environment at /Users/pelson/miniconda/envs/m4_clean:
#
m4                        1.4.17                        1    conda-forge
> /Users/pelson/miniconda/envs/m4_clean/bin/m4 --help
Illegal instruction: 4
> echo $?
132
```

(note: I have activated the environment, just to be certain)

```
 conda info
Current conda install:

               platform : osx-64
          conda version : 4.3.29
       conda is private : False
      conda-env version : 4.3.29
    conda-build version : 2.1.18
         python version : 3.5.4.final.0
       requests version : 2.18.4
       root environment : /Users/pelson/miniconda  (writable)
    default environment : /Users/pelson/miniconda/envs/m4_clean
       envs directories : /Users/pelson/miniconda/envs
                          /Users/pelson/.conda/envs
          package cache : /Users/pelson/miniconda/pkgs
                          /Users/pelson/.conda/pkgs
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
            config file : /Users/pelson/.condarc
             netrc file : /Users/pelson/.netrc
           offline mode : False
             user-agent : conda/4.3.29 requests/2.18.4 CPython/3.5.4 Darwin/17.2.0 OSX/10.13.1    
                UID:GID : 501:20
```

Any other OSX users seeing this? Related to High Sierra?

**Comment by brey** [comment]
Same problem here. 

Current conda install:

               platform : osx-64
          conda version : 4.3.29
       conda is private : False
      conda-env version : 4.3.29
    conda-build version : not installed
         python version : 3.6.3.final.0
       requests version : 2.18.4
       root environment : /Users/brey/miniconda3  (writable)
    default environment : /Users/brey/miniconda3/envs/mpi2
       envs directories : /Users/brey/miniconda3/envs
                          /Users/brey/.conda/envs
          package cache : /Users/brey/miniconda3/pkgs
                          /Users/brey/.conda/pkgs
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.continuum.io/pkgs/main/osx-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/osx-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/osx-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/osx-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : /Users/brey/.condarc
             netrc file : None
           offline mode : False
             user-agent : conda/4.3.29 requests/2.18.4 CPython/3.6.3 Darwin/17.3.0 OSX/10.13.2    
                UID:GID : 501:20

**Comment by brey** [comment]
version 1.4.18 from defaults works

**Comment by pelson** [comment]
Fixed in at least ``m4-1.4.18-h470a237_1``.
